test/WebJobs.Script.Tests.Analyzers/WebJobs.Script.Tests.Analyzers.csproj

```diff
@@ -7,6 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.33" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="4.0.4" />
```

Comment:
do we need this storage ref here?

Reply:
Not for this analyzer, no. Thanks for the catch.